### DISCUSSION TITLE
Can Audiogridder be used for remote monitoring?

### DISCUSSION THREAD

**Discussion by soulcook** [apohl79/audiogridder#754] [Q&A]
Hi

I have a DAW session running on a remote laptop and I need to monitor the audio from it on the computer in my studio. If I insert an AG plugin on the stereo bus in the laptop DAW and run AG Server in the studio, will it allow me to listen to the output of the remote DAW? 

Thanks!

Paul

**Comment by apohl79** [comment]
AG does not support this as the audio does not hit the audio device on the server side.

**Comment by soulcook** [comment]
Thank you.
